extensions-core/kafka-extraction-namespace/src/main/java/io/druid/query/lookup/KafkaLookupExtractorFactory.java

```diff
@@ -180,7 +181,7 @@ public boolean start()
       kafkaProperties.setProperty("group.id", factoryId);
       final String topic = getKafkaTopic();
       LOG.debug("About to listen to topic [%s] with group.id [%s]", topic, factoryId);
-      final Map<String, String> map = cacheManager.getCacheMap(factoryId);
+      final Map<String, String> map = cacheManager.getCacheMap(factoryId, KeyValueMap.DEFAULT_MAPNAME);
```

Comment:
Let's add a note in the docs about how KafkaLookupExtractor only uses the default mapname